src/templates/next/layouts/Collection/CollectionClient.tsx

```diff
@@ -210,33 +271,71 @@ const CollectionClient = ({
+
+      <div className="w-full sm:w-3/4 mx-auto">
         <CollectionSearch
           placeholder={`Search for ${page.title.toLowerCase()}`}
           search={searchValue}
           setSearch={setSearchValue}
         />
       </div>
-      <div className="flex flex-col sm:flex-row gap-10 justify-between w-full">
-        <div className="w-full sm:w-1/6">
+
+      <div className="flex flex-col lg:flex-row gap-10 justify-between w-full">
+        <div className="w-full lg:w-1/5 xl:w-1/6">
           <Filter
             filters={filters}
             appliedFilters={appliedFilters}
-            setAppliedFilters={setAppliedFilters}
+            setAppliedFilters={(id: string, itemId: string) =>
+              updateAppliedFilters(
+                appliedFilters,
+                setAppliedFilters,
+                id,
+                itemId,
+              )
+            }
           />
         </div>
-        <div className="flex flex-col gap-6 w-full sm:w-5/6">
-          <div className="flex justify-between w-full items-end">
-            <p className={`${Paragraph[1]} text-content`}>
-              {appliedFilters.length > 0 || searchValue !== ""
-                ? `${filteredItems.length} search ${
-                    filteredItems.length === 1 ? "result" : "results"
-                  }`
-                : `${items.length} ${
-                    items.length === 1 ? "article" : "articles"
-                  }`}
-            </p>
-            <div className="w-[260px]">
+        <div className="flex flex-col gap-6 w-full lg:w-4/5 xl:w-5/6">
+          <div className="flex flex-wrap sm:flex-nowrap justify-between w-full items-start gap-6">
+            <div className="flex flex-col gap-3 w-full h-full">
+              <p className={`${Paragraph[1]} text-content mt-auto`}>
+                {appliedFilters.length > 0 || searchValue !== ""
+                  ? `${filteredItems.length} search ${
+                      filteredItems.length === 1 ? "result" : "results"
+                    }`
+                  : `${items.length} ${
+                      items.length === 1 ? "article" : "articles"
+                    }`}
+                {searchValue !== "" && (
+                  <>
+                    {" "}
+                    for "<b>{searchValue}</b>"
+                  </>
+                )}
+              </p>
+
+              {appliedFilters.length > 0 && (
+                <div className="flex flex-row flex-wrap gap-3">
+                  {getAppliedFiltersWithLabels(filters, appliedFilters).map(
+                    (appliedFilter) => (
+                      <Pill
+                        key={`${appliedFilter.appliedFilterTypeId}-${appliedFilter.appliedFilterId}`}
+                        content={appliedFilter.label}
+                        onClose={() =>
+                          updateAppliedFilters(
+                            appliedFilters,
+                            setAppliedFilters,
```

Comment:
hmm why the api so powerful ah? by right it should just remove itself right?

Reply:
Yup it removes itself, this is just passing the arrow function into the Pill component which will only remove itself.